tools/podmanlinkchecker.sh

```diff
@@ -19,4 +19,9 @@ elif command -v docker
 else echo "No installation of podman or docker found in the PATH" ; exit 1
 fi
 
-${RUNNER} exec -ti che-docs "./tools/linkchecker.sh"
+# ${RUNNER} exec -ti che-docs "./tools/linkchecker.sh"
```

Comment:
we don't remove linkchecker tool if we use htmltest instead ?

Reply:
Correct, I should rename the script

Reply:
Removed linkchecker scripts